timemachine/fe/single_topology.py

```diff
@@ -1165,15 +1289,14 @@ def setup_intermediate_state(self, lamb) -> VacuumSystem:
         Note that the above only applies to the interactions whose force constant is zero in one end state; otherwise,
         valence terms are interpolated simultaneously in the interval :math:`0 \leq \lambda \leq 1`)
         """
-
         src_system = self.src_system
         dst_system = self.dst_system
 
         # stagger the lambda schedule
-        bonds_min, bonds_max = [0.0, 0.3]
-        angles_min, angles_max = [0.6, 0.8]
-        torsions_min, torsions_max = [0.8, 1.0]
-        chiral_atoms_min, chiral_atoms_max = [0.3, 0.6]
+        bonds_min, bonds_max = [0.0, 0.7]
+        angles_min, angles_max = [0.0, 0.7]
+        torsions_min, torsions_max = [0.7, 1.0]
+        chiral_atoms_min, chiral_atoms_max = [0.7, 1.0]
```

Comment:
Nice: intuitively, this seems more efficient.

(if these transformations no longer conflict with each other very much, seems more economical to schedule them simultaneously)